app/src/main/res/layout/layout_code_validation.xml

```diff
@@ -7,190 +7,199 @@
     android:focusable="true"
     android:focusableInTouchMode="true"
     >
-  <androidx.constraintlayout.widget.ConstraintLayout
-      android:id="@+id/content"
+
+  <ScrollView
```

Comment:
This ScrollView should contain both the "layout_code_validation" and the progress animation